packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.constant.js

```diff
@@ -271,13 +271,19 @@ export const LEGAL_LINK3 = {
 
 export const MAX_SIZE = 10000000;
 
-export const TRACKING_PREFIX = 'dedicated::account::identity-files';
+export const TRACKING_PREFIX = 'Hub::account::identity-files';
```

Comment:
I guess it was asked this way (so this comment is not blocking), but I think this prefix should be removed entirely, as 'Account' module will be extracted from dedicated (and is in no way part of the hub module as suggested by this prefix)